src/lib/y2partitioner/widgets/btrfs_subvolumes_table.rb

```diff
@@ -115,7 +116,12 @@ def path_help
 
       def nocow_help
         _("<b>noCoW</b> shows the subvolume noCoW attribute. " \
-          "If set, the subvolume is not copy on write.")
+          "If set, the subvolume explicitly does not use Btrfs copy on write feature. " \
+          "Copy on write means that when something is copied, the resource is shared without " \
+          "doing a real copy. The shared resource is actually copied when first write operation " \
+          "is performed. With noCoW, the resource is always copied during initialization. " \
+          "This is useful when runtime performace is required, and there is no risk for delaying " \
```

Comment:
I think it need someone with better english skill, but at least I replace ", and" with ", so "